core/lib/dal/migrations/20240723143707_remove_redundant_keys.up.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE protocol_versions DROP COLUMN IF EXISTS recursion_node_level_vk_hash;
+ALTER TABLE protocol_versions DROP COLUMN IF EXISTS recursion_leaf_level_vk_hash;
+ALTER TABLE protocol_versions DROP COLUMN IF EXISTS recursion_circuits_set_vks_hash;
+
+ALTER TABLE protocol_patches DROP COLUMN IF EXISTS recursion_node_level_vk_hash;
+ALTER TABLE protocol_patches DROP COLUMN IF EXISTS recursion_leaf_level_vk_hash;
+ALTER TABLE protocol_patches DROP COLUMN IF EXISTS recursion_circuits_set_vks_hash;
```

Comment:
This is breaking migration. You can include it only in the release after the next one